sdk/server/logger.go

```diff
@@ -0,0 +1,77 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/x-cray/logrus-prefixed-formatter"
+)
+
+// Logger represents a generic logger, based on logrus.Logger
+type Logger interface {
+	Debugf(format string, args ...interface{})
+	Errorf(format string, args ...interface{})
+	Fatalf(format string, args ...interface{})
+	Infof(format string, args ...interface{})
+	Warningf(format string, args ...interface{})
```

Comment:
Why the `f`'s at the end?

Reply:
because it is `Printf` format, and as is written in the doc, it comes from logrus.Logger struct